CMakeLists.txt

```diff
@@ -2,25 +2,40 @@
 # Copyright (c) 2017 Michael Heimpold <mhei@heimpold.de>
 #
 
-cmake_minimum_required (VERSION 2.8)
-project (openv2g C)
+cmake_minimum_required(VERSION 3.14)
 
-set(OPENV2G_MAJOR_VERSION 0)
-set(OPENV2G_MINOR_VERSION 9)
-set(OPENV2G_PATCH_VERSION 5)
-set(VERSION ${OPENV2G_MAJOR_VERSION}.${OPENV2G_MINOR_VERSION}.${OPENV2G_PATCH_VERSION})
-set(SOVERSION 1)
```

Comment:
SOVERSION is used in lib/CMakeLists.txt i don't think this should be removed

Reply:
Correct, i changed that back